amazonorders/selectors.py

```diff
@@ -1,6 +1,8 @@
 __copyright__ = "Copyright (c) 2024 Alex Laird"
 __license__ = "MIT"
 
+from amazonorders.constants import Constants
```

Comment:
`Constants` always needs to be access through `self.config.constants`, otherwise it'll beak if someone overrides the class. Remove this import.